integration-tests/src/test/resources/integration-test/distro-properties/openmrs-distro.properties

```diff
@@ -0,0 +1,8 @@
+#Fri Dec 30 10:54:43 CET 2016
+version=2.5
+name=Reference Application
+distro.referenceapplication.artifactId=referenceapplication-package
```

Comment:
Fixed
Its exclusive feature for `referenceapplication` since I didn't see any other distro with `-package` suffix